package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "bayan_ali--project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "nodemon src",
+    "test": "jest",
+    "start": "node src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CA-G12/Bayan_Ali--project.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/CA-G12/Bayan_Ali--project/issues"
+  },
+  "homepage": "https://github.com/CA-G12/Bayan_Ali--project#readme",
+  "dependencies": {
+    "compression": "^1.7.4",
+    "dotenv": "^16.0.1",
+    "express": "^4.18.1",
+    "jest": "^28.1.3",
+    "node-fetch": "^2.6.7",
+    "nodemon": "^2.0.19",
+    "supertest": "^6.2.4"
```

Comment:
some packages like `nodemon` is used for development sake, so it would make more sense to install them as a devdependencies.

https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/